src/converters/geoJSONConverter.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+let logger = require('logger');
+
+module.exports.convert = function(data){
+    if(data.coordinates){
```

Comment:
Should we be using the `type` attribute here, instead? e.g. `if (data.type === 'FeatureCollection') { ... } else if (data.type === 'Feature') { ... } else { // handle geometries }`